src/array.rs

```diff
@@ -0,0 +1,184 @@
+//! ## Array representation
+//! Allows to estimate medium cardinality in [3..MAX_CAPACITY] range.
+//!
+//! The `data` format of array representation:
+//! - 0..1 bits     - store representation type (bits are set to `01`)
+//! - 2..55 bits    - store pointer to `u32` slice (on `x86_64 systems only 48-bits are needed).
```

Comment:
nit: unclosed backtick.
```suggestion
//! - 2..55 bits    - store pointer to `u32` slice (on `x86_64` systems only 48-bits are needed).
```

Reply:
Fixed.